src/ui/views/ui_app.jade

```diff
@@ -27,7 +27,7 @@ block content
     h1
       strong Kibana
```

Comment:
Do either of you have an idea of how to solve this without allowing the translators to use html?